packages/components/src/components/search/search.tsx

```diff
@@ -319,6 +337,46 @@ export class Search implements ComponentInterface {
         });
     }
 
+    /**
+     * Get Mapbox Places results.
+     *
+     * @param {string} query
+     * @return {Promise<any>}
+     */
+    private getMapboxSearchResults(query: string): Promise<any> {
+        if (this.mapbox && this.mapboxAccessToken) {
+            if (query) {
+                return new Promise((resolve) => {
+                    const url = `https://api.mapbox.com/search/searchbox/v1/suggest?q=${query}&session_token=[GENERATED-UUID]&access_token=${this.mapboxAccessToken}`;
```

Comment:
I guess we need a proper `session_token` here.

Reply:
Yes, this is in progress.